securityscanutils/trivy_scanner.go

```diff
@@ -16,6 +18,8 @@ import (
 // Status code returned by Trivy if a vulnerability is found
 const VulnerabilityFoundStatusCode = 52
 
+var ImageNotFoundError = errors.New("❗IMAGE MISSING UNEXPECTEDLY❗")
```

Comment:
Will the emojis show up properly in GitHub? Do we need ":exclamation:" instead?

Reply:
https://github.com/t-edris/testing-things

It looks like the actual emoji's do show up and `:exclamation:` does in some cases. I don't understand the rules for that quite yet as evidenced by the commit history there.

It seems most consistent to keep this as-is, since that seems to always render.

Reply:
Nice exploration, thanks for checking it out! Sounds good.